blackbox/docs/general/ddl/alter-table.rst

```diff
@@ -32,6 +32,35 @@ In order to set a parameter to its default value use ``reset``::
     cr> alter table my_table reset (number_of_replicas);
     ALTER OK, -1 rows affected (... sec)
 
+.. _alter_change_number_of_shard:
+
+Changing Number of Shards
+-------------------------
+
+It is possible to change the number of primary shards of a table
+
+..    todo: remove comments once feature is implemented
+..    cr> alter table my_table set (number_of_shards = 2);
+..    ALTER OK, -1 rows affected (... sec)
```

Comment:
We want this will to fail until implementation is done. (enable it, remove comments)